requirements/reproducible-win.txt

```diff
@@ -0,0 +1,785 @@
+#
```

Comment:
Is this file different from the MacOS file? Looks like the command to create it is the same. Does it need to be run on Windows?

Reply:
Also, if this should be run on a Windows machine, maybe we need to move it to a GHA?